eahub/localgroups/urls.py

```diff
@@ -18,4 +19,14 @@
         views.LocalGroupDeleteView.as_view(),
         name="localgroups_delete",
     ),
+    urls.path(
+        '<slug:slug>/report-abuse/',
+        views.report_abuse,
+        name='report_abuse_group'
+    ),
+    urls.path(
+        '<slug:slug>/report-abuse-done/',
+        TemplateView.as_view(template_name='eahub/report_abuse_done.html'),
```

Comment:
Would it maybe be better for this to redirect back somewhere useful with a flash message instead of having a separate landing page?

Reply:
Yes, good idea.Redirect to the profile that is reported on and added flash message there.